File: scratch/check_triggers.js
const mysql = require('mysql2/promise');

async function main() {
  const pool = mysql.createPool({
    host: '172.17.3.206',
    user: 'secom',
    password: 'secom123',
    database: 'secom',
    port: 3306
  });

  try {
    const [triggers] = await pool.query("SELECT * FROM t_secom_trigger");
    console.log(triggers);
  } catch (err) {
    console.error(err);
  } finally {
    await pool.end();
    process.exit();
  }
}

main();

File: scratch/check_triggers.py
import pymysql

try:
    conn = pymysql.connect(host='172.17.3.206', user='secom', password='secom123', db='secom')
    cursor = conn.cursor()
    
    cursor.execute("SELECT id, function_name, event_source, time_type, time_value, is_active, days_of_week, receivers FROM t_secom_trigger")
    rows = cursor.fetchall()
    print("--- Triggers ---")
    for r in rows:
        print(r)
        
except Exception as e:
    print(f"Error: {e}")
